wagtail_live/models.py

```diff
@@ -59,6 +58,168 @@ class LivePageMixin(models.Model):
         StreamFieldPanel("live_posts"),
     ]
 
+    def _get_live_post_index(self, live_post_id, low, high):
+        """ Retrieve index of a live post in page's live posts.
+
+        Uses binary search since live posts are sorted by timestamp.
+        This implementation may change.
```

Comment:
Binary search seems a bit overkill for a page where we don't expect that many posts :-). If there would be so many, say more than 100, posts, we would rather need a [load more] functionality for the user first I'd say. But I like the effort! Also, binary search is built-in in python if you use the bisect module!